src/api/models/Activity.ts

```diff
@@ -25,12 +25,16 @@ import {
 import sequelize from '../../config/sequelize';
 import { Genre } from './Genre';
 import { Label } from './Label';
+import {Address} from "./Address";
 import { ActivityReview } from './ActivityReview';
```

Comment:
You should either have a space after/before the curly braces or none. This is for consistency. This could be done with lint